### ISSUE TITLE
Not dimming in certain cases

### ISSUE THREAD

**Issue by haji-ali** [mina86/auto-dim-other-buffers.el#22]
I have the following code to enable `auto-dim-other-buffers`

```
(add-hook 'after-init-hook (lambda ()
  (when (fboundp 'auto-dim-other-buffers-mode)
    (auto-dim-other-buffers-mode t))))

(set-face-attribute 'auto-dim-other-buffers-face nil :background "#111111")
```

I noticed two cases where the inactive windows are not dimmed (assume my Emacs has two open windows):
- When Emacs loses focus, both windows are dimmed, but when I return to Emacs none of the windows is dimmed until I call `other-window`.
- If I have two open windows `Win1` and `Win2`. Start with focus in `Win1`, then call, for example, `decribe-variable` to describe some variable. `Win2` then shows the variable description and is focused while `Win1` is correctly dimmed. However, if I kill the buffer by pressing `q` then the focus returns to `Win1` but `Win2` is *not* dimmed.

I am running Emacs 26.3 and version 2.0.1 of `auto-dim-other-buffers`.

**Comment by mina86** [comment]
The second issue should now be resolved with the 2.0.2 which I’ve just pushed.

The first issue, however, I am unable to reproduce. Does it always happen?
This may be related to [focus management fixes](https://git.savannah.gnu.org/cgit/emacs.git/commit/?id=2f6c682061a281dc3e397ff4727a164880e86e7b) which I don’t believe made it to 26.3 but are only included in 27.

**Comment by haji-ali** [comment]
I just pulled and the second issue seems fixed. Thanks!

For the first issue, it is pretty consistent on my setup. I will update to Emacs 27 to check if it persists.
